sample_files/pre-commit-13.0/.pre-commit-config.yaml

```diff
@@ -0,0 +1,62 @@
+exclude: "^setup/"
+default_language_version:
+  python: python3
+repos:
+- repo: https://github.com/psf/black
+  rev: 19.3b0
+  hooks:
+  - id: black
+- repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
There are a few more hooks that seem elegible for our case. What do you think?:

1. check-case-conflict
2. check-docstring-first
3. check-executables-have-shebangs
4. check-merge-conflict
5. check-symlinks
6. check-xml
7. mixed-line-ending

Reply:
Yes, makes sense. `fix-encoding-pragma --remove` sounds interesting too.

Reply:
Done. I chose `--fix=lf` for `mixed-line-ending`.